blank_react_project/assembly/main.ts

```diff
@@ -0,0 +1,15 @@
+import { near, context, storage, logging, base58, base64, PersistentMap, PersistentVector, PersistentDeque, PersistentTopN, ContractPromise, math } from "near-runtime-ts";
+// import { u128 } from "bignum";
+// import { TextMessage } from "./model";
+
+const NAME = ". Welcome to NEAR Protocol chain"
+
+export function hello(name: string): string {
```

Comment:
looks like we have both 2-space and 4-space indentation in different places of PR. I think we should stick to same indent size everywhere for simplicity, 4-space seems to match most of our stuff (unless anyone has objections)